src/xcode/ENA/ENA/Source/Client/HTTP Client/HTTPClient+Configuration.swift

```diff
@@ -39,6 +39,7 @@ extension HTTPClient {
 
 			return Configuration(
 				apiVersion: "v1",
+				encryptedApiVersion: "v2",
```

Comment:
I think if we refactor the http stuff, we should take into account that not every endpoint will operate with the same version. Like we already see with the encrypted API.